cli/src/main.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright 2018 Bitwise IO
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#[macro_use]
+extern crate clap;
+#[macro_use]
+extern crate serde_derive;
+extern crate dirs;
+extern crate toml;
+
+mod commands;
+mod utils;
+
+use clap::App;
+use commands::init;
+
+fn main() {
+    let yaml = load_yaml!("cli.yaml");
+    let args = App::from_yaml(yaml).get_matches();
+
+    let result = match args.subcommand() {
+        ("init", Some(args)) => init::run(args),
+        _ => {
+            println!("Invalid subcommand; pass --help for usage.");
```

Comment:
Should this be `eprintln!` as well? I think it is the recommended macro fro printing errors

Reply:
Good point. Instead, I added a setting which just prints the --help when you call `bond-cli`, and changed this match function to `unreachable!()` since clap will handle the case with no subcommand.